components/ws-manager/pkg/manager/manager.go

```diff
@@ -808,7 +816,7 @@ func (m *Manager) StopWorkspace(ctx context.Context, req *api.StopWorkspaceReque
 
 // stopWorkspace stops a workspace. If the workspace is already stopping, this is a noop
 func (m *Manager) stopWorkspace(ctx context.Context, workspaceID string, gracePeriod time.Duration) (err error) {
-	if m.Config.DryRun {
+	if m.Config.MaintenanceMode {
 		log.WithFields(log.OWI("", "", workspaceID)).Info("should have stopped pod but this is a dry run")
```

Comment:
Should this message return under maintenance mode, too, rather than dry run message?